### ISSUE TITLE
[BUG/ISSUE] Planeflight_mod.F90 MaxRO2 value needs to be updated 

### ISSUE THREAD

**Issue by jdhask** [geoschem/geos-chem#659]
Similar to [Issue #397]( https://github.com/geoschem/geos-chem/issues/397).

I was getting a persistent error: "NPRO2 exceeds maximum allowed value. Stop in RO2_Setup (planeflight_mod.F90)" while running a v13.0.0 complex SOA with semivolatile POA fullchem simulation and trying to sample along a flight track. 

The variable MAXRO2 was set to a value of 45 on line 98, but I think it should be set to 56. There are 56 species on lines 1200-1213 that are considered to be an RO2 component. I was only using 46 of them in my run (hence tripping the error), but after updating line 98 to make maxRO2= 56 to match the list  it's checked against, I was able to complete the run without problems. 

! Parameters  ... Line 98: 
~~INTEGER,           PARAMETER   :: MAXRO2    = 45~~
  INTEGER,           PARAMETER   :: MAXRO2    = 56

Lines 1135-1150: 
```
       ! If we have found an RO2 compoent, add its species # to
       ! the PRO2 global array, and increment counter
       ! NOTE: PO3 was a bug, that should have been PO2 (tmf, 2/10/09)
       SELECT CASE( TRIM( SpcInfo%Name ) )

       CASE ( 'HO2',     'MO2',      'A3O2',     'ATO2',    'B3O2',  &
              'ETO2',    'HPALD1OO', 'HPALD2OO', 'ICHOO',            &
              'GCO3',    'IAO2',     'KO2',      'MCO3',    'PO2',   &
              'ACO3',    'EO2',      'ENCO3',    'ENO2',    'GLCO3', &
              'ICNOO',   'IDHNBOO',  'IDHNDOO1', 'IDNOO',            &
              'IDHNDOO2', 'IEPOXAOO', 'IEPOXBOO',                     &
              'IHOO1',   'IHOO4',    'IHPNBOO',  'IHPNDOO',          &
              'IHPOO1',  'IHPOO2',   'IHPOO3',   'INO2B',            &
              'INO2D',   'ISOPNOO1', 'ISOPNOO2', 'LIMO2',            &
              'MACR1OO', 'MACRNO2',  'MCROHOO',  'PIO2',             &
              'MVKOHOO', 'R4O2',     'R4N1',     'R4N2',             &
              'C4HVP1',  'C4HVP2',                                   &
              'BRO2',    'TRO2',     'XRO2',     'NRO2',             &
              'NICO3',   'NIO2',     'PYPO2',    'RCO3')
          NPRO2       = NPRO2 + 1
          PRO2(NPRO2) = M
```

**Comment by msulprizio** [comment]
Thanks for reporting this issue @jdhask. We can include this fix in 13.0.1.

**Comment by msulprizio** [comment]
This is now fixed in commit https://github.com/geoschem/geos-chem/commit/0727622b7e7f32bf022f27806661812c48a56543 and will be included in 13.0.1, which we should release in the next few days.
